<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>601</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;SmartDiff&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Step 5: This is your result&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ValLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Function value&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="DerLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Function derivative &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;(Jacobian matrix)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="f3Label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>410</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;f3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f1Val">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="default is 0"/>
   </property>
   <property name="placeholderText">
    <string extracomment="Default is 0"/>
   </property>
  </widget>
  <widget class="QLabel" name="f1Label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;f1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="f2Label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;f2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f3Val">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="default is 0"/>
   </property>
   <property name="placeholderText">
    <string extracomment="Default is 0"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="f2Val">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="default is 0"/>
   </property>
   <property name="placeholderText">
    <string extracomment="Default is 0"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="f11Der">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="f12Der">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="f13Der">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>290</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="f11Label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>df1/dx1</string>
   </property>
  </widget>
  <widget class="QLabel" name="f12Label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>df1/dx2</string>
   </property>
  </widget>
  <widget class="QLabel" name="f13Label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>270</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>df1/dx3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f21Der">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="f23Label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>df2/dx3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f23Der">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>360</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="f22Label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>df2/dx2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f22Der">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>360</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="f21Label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>df2/dx1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f31Der">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>430</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="f32Label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>410</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>df3/dx2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f32Der">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="f33Der">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="f31Label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>df3/dx1</string>
   </property>
  </widget>
  <widget class="QLabel" name="f33Label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>410</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>df3/dx3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrMsg">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>571</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
